numpy/random/common.pxd

```diff
@@ -5,7 +5,7 @@ from libc.stdint cimport (uint8_t, uint16_t, uint32_t, uint64_t,
                           uintptr_t)
 from libc.math cimport sqrt
 
-cdef extern from "numpy/random/bitgen.h":
+cdef extern from "src/bitgen.h":
```

Comment:
The original is what is in 1.17 and will also be in 1.18 after the C-API changes go in.